composer.json

```diff
@@ -2,6 +2,7 @@
     "name": "mtudor/zfc-user-impersonate",
     "description": "Facilitate impersonation of other users in ZfcUser.",
     "type": "library",
+    "license": "LGPL-2.1",
```

Comment:
Hey @reiz - thanks for this; apologies for the record-breaking response time!  Two quick questions and then we can get this merged:

1. It looks like the value of this should be "LGPL-2.1-or-later" rather than just "LGPL-2.1"? https://spdx.org/licenses/

2. Out of interest. what prompted you to need the license in composer rather than the LICENSE file; does it have a real world impact or are you just tidying up?

Thanks!